src/presentation/controllers/ListCustomersController.ts

```diff
@@ -0,0 +1,26 @@
+import { tokens } from "@/di/tokens";
+import { Response, Request } from "express";
+import { IRouterController } from "@/presentation/routes/Controller";
+import httpStatus from "http-status";
+import { inject, injectable } from "tsyringe";
+import { ICustomerService } from "@/domain/customer/types/services/ICustomerService";
+
+@injectable()
+export class ListCustomerController implements IRouterController {
+  constructor(
+    @inject(tokens.CustomerService)
+    private customerService: ICustomerService
+  ) {
+    this.customerService = customerService;
```

Comment:
Essa linha é necessária? Não daria para chamar o this.customerService direto na linha 18?